simulatorUser/use.rst

```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
 * ``even``: every other fiber, starting with the second.
 * ``all``: all fibers.
 
-The targets may be:
+``DRP_INSTDATA_DIR``
+~~~~~~~~~~~~~~~~~~~~
 
-* ``Arc``: an arc spectrum with all lamps for all fibers.
-* ``Ne``: an arc spectrum with the Ne lamp for all fibers.
-* ``Hg``: an arc spectrum with the Hg lamp for all fibers.
-* ``Flat``: a quartz spectrum for all fibers.
-* ``Comb``: a set of emission lines of equal spacing for all fibers.
-* ``Const``: a constant :math:`F_nu` spectrum for all fibers.
-* ``Sky``: a pure sky spectrum for all fibers.
-* ``Obj``: a combination of sky, flux calibration sources and objects.
+Also before running the simulator the envionment variable ``DRP_INSTDATA_DIR`` needs to be set to point
+to a recent version of the ``drp_instdata`` git repository. For example::
+
+    cd /path/to
+    git clone https://github.com/Subaru-PFS/drp_instdata.git
+    export DRP_INSTDATA_DIR=/path/to/drp_instdata
```

Comment:
Better to `setup -jr /path/to/drp_instdata`.
Be warned that drp_instdata is a large package.

Reply:
Good point. Fixed.